sdk/python/packages/flet-core/src/flet_core/control.py

```diff
@@ -337,6 +337,9 @@ def invoke_method(
         assert (
             self.__page
         ), f"{self.__class__.__qualname__} Control must be added to the page first"
+        if arguments:
+            # remove items with None values and convert other values to string
+            arguments = {k: str(v) for k, v in arguments.items() if v is not None}
         return self.__page._invoke_method(
```

Comment:
What issue does it solve?

Reply:
I'm not sure it won't break anything. I can be mistaken, but there could be some controls expecting bool or int on dart side from args JSON.

Reply:
It's a while now, but i remember it was to always make sure values are always sent over as string (as required by definition: `arguments: Optional[Dict[str, str]]`). Kind of a pydantic-like validation process.
I had equally notice that sending `None` values to dart end lead to errors, so they are removed before being sent.

> I can be mistaken, but there could be some controls expecting bool or int on dart side from args JSON.

Checkout other controls like `Audio` to notice that we send everything as string no matter the type.
On dart end they are handled accordingly with utils like `parseDouble()`

Reply:
Any update

Reply:
@Muddassir-Farooq-official please stop spamming! It is really annoying. 
Simply be patient and give us time to work.